Neos.ContentRepository.Core/Classes/CommandHandler/CommandHooks.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core\CommandHandler;
+
+/**
+ * Collection of {@see CommandHookInterface} instances
+ *
+ * @implements \IteratorAggregate<CommandHookInterface>
+ * @internal
+ */
+final readonly class CommandHooks implements \IteratorAggregate, \Countable
```

Comment:
maybe a delegating `CommandHook` would be less boiler plate code :D

Reply:
mh yes, maybe but it's also another level of indirection.. not sure

Reply:
I decided to keep the explicit `CommandHooks` but make it implement the `CommandHookInterface` so that it can be used as hook directly with a3aa2266bb5d77fa55b64e6a76ca09aeb18c6e14

Reply:
...but I relaxed the CR type hint with ccbe354276ed941ccb70882b3d658bd585b47941